packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -235,12 +235,6 @@ export const GRID_ROW_LEAVE = 'rowLeave';
  */
 export const GRID_ROW_EDIT_MODEL_CHANGE = 'editRowModelChange';
 
-/**
- * Fired when a row is selected or unselected. Called with a [[GridRowSelectedParams]] object.
- * @event
- */
-export const GRID_ROW_SELECTED = 'rowSelected';
```

Comment:
👍 I remember we discussed removing this event. Just need to remember to add it in the breaking changes list.